### ISSUE TITLE
Got black screen in VMware® Workstation 11.1.3 build-3206955

### ISSUE THREAD

**Issue by eqvis** [MicrosoftDocs/sysinternals#8]
Perhaps a message about this problem could be added in the documentation (could produce black screen on start up of VMware clients!?) Didn't know where to tell this case. (Both machines are Windows 7). Worked again after reboot of the host with disabled Desktops.

VMware Log:

2017-09-04T09:15:36.019+02:00| mks| I120: MKS-SWB: Window #0 validation failed: no valid host window or host surface.
2017-09-04T09:15:36.019+02:00| mks| I120: GDI-Backend: successfully started by HWinMux to do window composition.
2017-09-04T09:15:36.019+02:00| mks| I120: MKS-HWinCompMux: Started GDI presentation backend
2017-09-04T09:15:36.019+02:00| mks| W110: MKSWin32_CreateWindow: Failed to create window: Unknown error 1400 (0x578) (1400)
2017-09-04T09:15:36.019+02:00| mks| I120: MKS-HWinCompMux: Failed PreDefineWindow
2017-09-04T09:15:36.019+02:00| mks| W110: MKS-SWB: HWin failed to define window.
2017-09-04T09:15:36.019+02:00| mks| W110: MKSWin32_CreateWindow: Failed to create window: Unknown error 1400 (0x578) (1400)
2017-09-04T09:15:36.019+02:00| mks| I120: MKS-HWinCompMux: Failed PreDefineWindow
2017-09-04T09:15:36.019+02:00| mks| W110: MKS-SWB: HWin failed to define window.
2017-09-04T09:15:36.035+02:00| vmx| I120: Vix: [5788 mainDispatch.c:4790]: VMAutomationProcessMessage: Postpone the command. MSG in progress (opcode 151)

Best Regards,
Mayra

**Comment by lukekim** [comment]
Thanks for filing this issue. To help us investigate, can you please:

- Make sure you have the latest version of Desktops & VMWare integration components
- Reproduced the issue on multiple VMs that are clean Windows installs (no third party code)
- Provide detailed reproduction steps

Thanks,
Luke

**Comment by eqvis** [comment]
Hello Luke,

I’m very sorry. I found this at work and there I have no administrative rights (cannot update the Software). And at home I do not use VMware and have no other Windows Licences.

What I did was download the current Desktops.exe (last month) and run it after Login. Then I startet the VMware Workstation and the Client (Windows 7). It Looks like it starts, but then it remains black, appearing only scroll bars.

If I can test something at work, I will send you more Information. The next 2 Weeks I’m on vacances.



Best regards,
Mayra
